string node_name
string name
string default_value
---
string value